src/main/webapp/paginationScript.js

```diff
@@ -61,11 +61,14 @@ function displayCards(listAugment) {
       //Card being appended to the resultsContent div
       let cardToAppend = resultsCardsArray[i];
 
+      //The actual image element to which the image src will be applied
+      let imageElement = cardToAppend.card.children[0].children[0]; 
```

Comment:
Looks like this is following the append order on line 134 in script.js. 
Can you incorporate the classname  'results-image' here?

Reply:
Done

Reply:
Was this updated?

Reply:
It was, I changed imageElement to resultsImageElement - is this generally what you meant?

Reply:
Not quite. I meant instead of doing "cardToAppend.card.children[0].children[0]" can you find the image element given its class name? That way this is dependent on what order the elements are appended in in script.js.